ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -164,6 +170,52 @@ protected boolean setupStrokePaint(Paint paint, float opacity) {
     return true;
   }
 
+  /*
+   * calculate stops count from array
+   * from array minus first 5 elements because it 1 - LinearGradient key, 2,3,4,5 - LinearGradient position
+   * and after devide on 5 because 1,2,3,4 - colors A,R,G,B and 5 - stop per 4 colors.
+   */
+  private static int getStopsCount(float[] value) {
+    int count = (value.length - 5)/5;
+    return count;
+  }
+
+  /*
+   * sorting stops and stopsColors from array
+   */
+  private static int gradientStopsParsing(float[] value, int stopsCount, float[] stops, int[] stopsColors) {
+    int startStops = value.length - stopsCount;
+    int startColorsPosition = 5;
+    for (int i = 0; i < stopsCount; i++) {
+      stops[i] = i == 0 ?
+        value[startStops + i] :
+        (
+          i == (stopsCount - 1) ? value[startStops + 1] :
+          value[startStops + i + 1]
+        );
+      stopsColors[i] = i == 0 ?
+        Color.argb(
+          (int) (value[startColorsPosition + 3] * 255),
+          (int) (value[startColorsPosition] * 255),
+          (int) (value[startColorsPosition + 1] * 255),
+          (int) (value[startColorsPosition + 2] * 255)) :
+        i == (stopsCount - 1) ?
+          Color.argb(
+            (int) (value[12] * 255),
```

Comment:
@dmmiller  There is another problem.
From react colors income like as R,G,B,A but Android accepting A,R,G,B)))